docs/api.md

```diff
@@ -184,6 +285,44 @@ Content:
 }
 ```
 
+##### Comment Not Found
+
+The `comment-id` parameter provided was not found.
+
+Code: 404
+
+Content:
+```
+{
+    "type": "object",
+    "properties": {
+        "message": {
+            "description": "Error message",
+            "type": "string"
+        }
+    }
+}
+```
+
+##### Rating Not Found
+
+The `rating-id` parameter provided was not found.
+
+Code: 404
+
+Content:
+```
+{
+    "type": "object",
+    "properties": {
+        "message": {
+            "description": "Error message",
+            "type": "string"
+        }
+    }
+}
+```
+
```

Comment:
Having one error specification per resource is perfectly fine, but I think they could be combined into one 'resource not found' error without any confusion. To minimize the confusion we could have "lecture, comment or rating was not found" in the description.